### PR TITLE
[Polish] Fix for launch screen being truncated

### LINKED ISSUE #2473: Launch image doesn’t show logo text when status bar frame is larger.
Reproduce:

1. Kill Eigen
2. Launch Maps and start a navigation route
3. Tap home button and ensure the status bar is now large and blue
3. Launch Eigen and notice text from logo is gone

img_1351

### PR DESCRIPTION
**Pull request by orta**
Fixes #2473 - I think

screen shot 2017-12-02 at 11 43 18 am

### CODE REVIEW

File: Artsy/App_Resources/Launch Screen.xib [hunk 1]
```diff
@@ -11,13 +15,10 @@
...
             <subviews>
                 <imageView opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleAspectFill" image="full_logo_white_large.png" translatesAutoresizingMaskIntoConstraints="NO" id="WeP-8T-eYW">
-                    <rect key="frame" x="259" y="469" width="250" height="86"/>
-                    <animations/>
+                    <rect key="frame" x="324.5" y="491.5" width="119" height="41"/>
                     <constraints>
-                        <constraint firstAttribute="width" relation="lessThanOrEqual" constant="250" id="J9h-cn-RfE"/>
                         <constraint firstAttribute="width" constant="119" id="LxL-2v-6xP"/>
                         <constraint firstAttribute="height" constant="41" id="VtU-kR-P6R"/>
-                        <constraint firstAttribute="height" relation="lessThanOrEqual" constant="86" id="qsT-5K-yhF"/>
```
Comment (orta) [inline Artsy/App_Resources/Launch Screen.xib]: It's a guess, but I can't see a reason for these constraints

Comment (alloy) [reply Artsy/App_Resources/Launch Screen.xib]: Could it have to do with iPad in landscape mode?

**Comment by orta** [comment]
Seems fine

screen shot 2017-12-04 at 7 13 55 am

**Comment by alloy** [comment]
👍

**Pull request merged**